debug.sh

```diff
@@ -7,32 +7,38 @@ VM_USERNAME="root"
 VM_PASSWORD="123456"
 VM_SNAPSHOT="first_snapshot"
 KERNEL_MODULE_NAME="main"
-
 CWD=`pwd`
 REMOTE_DIR="/root/"`basename $CWD`
 SSH_PORT=5555
 
 cd $BUILDROOT_IMAGES_PATH
 
-echo "Starting the vm"
-setsid qemu-system-x86_64 -enable-kvm -cpu host -s -kernel bzImage -m 2048 -hda rootfs.qcow2 -append "root=/dev/sda rw nokaslr" -net nic,model=virtio -net user,hostfwd=tcp::$SSH_PORT-:22 -loadvm $VM_SNAPSHOT &
+if pgrep "qemu" > /dev/null
```

Comment:
Really cool improvements !
In my opinion if the user press F5 in order to debug while qemu is alive we should kill it (qemu) and then perform the regular flow